products/green_fast_track/models/intermediate/flags/int_flags__dob_natural_resources.sql

```diff
@@ -0,0 +1,57 @@
+-- this file should maybe be broken up, as it's doing two things
+-- 1. supplying flags that go straight into int_flags__all
+-- 2. supplying geoms that go into natural_resource shadows
+-- 3. supplying geoms that go into natural_resource source outputs
```

Comment:
since in a comment below you said this model doesn't feed into `int_spatial__natural_resources` because it doesn't need a spatial join to determine flags since they're provided tabularly, who do the geoms need to go into Shadows Natural Resources?